src/core/include/mp-units/ext/prime.h

```diff
@@ -42,6 +42,86 @@ import std;
 
 namespace mp_units::detail {
 
+// (a + b) % n.
+//
+// Precondition: (a < n).
+// Precondition: (b < n).
+// Precondition: (n > 0).
```

Comment:
You can use `MP_UNITS_EXPECTS_DEBUG` if needed.

Reply:
I'd love to express these in code, but here is what I see:

![image](https://github.com/user-attachments/assets/81c13b7e-689b-4791-8015-113ce89a7fed)

This _is_ with `#include <mp-units/compat_macros.h>`.

Any advice for a quick fix?

Reply:
Sure 😉 

https://github.com/mpusz/mp-units/blob/dc6fb931ea342a21c76709e35f88b6d44ec9710d/src/core/include/mp-units/framework/quantity.h#L39-L40